plugins/flatpak/gs-flatpak.c

```diff
@@ -3244,6 +3263,30 @@ gs_flatpak_create_app_from_repo_dir (GsFlatpak *self,
 	return app;
 }
 
+static void
+gs_flatpak_mount_removed (GVolumeMonitor *volume_monitor,
+			  GMount *mount,
```

Comment:
Nitpick: Indentation problem with arguments.

Reply:
Although I’ve just realised that all these indentation problems may be caused by github not aligning the tabs the same way as your editor. If so, ignore my comments.